### PR TITLE
253 create a taskcard for the board page with tasks

### PR DESCRIPTION
**Pull request by VladDot**
Create a taskcard for the board page 
Implemented functional deletion for the task card
Added icon DotSettings
Created a temporary mock for the example

### CODE REVIEW

File: client/src/components/board-task-card/card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  setData: React.Dispatch<React.SetStateAction<IMockCards[]>>;
+}
+
+export const Card = ({ id, title, users, setData, data }: ICard) => {
+  const deleteMessage = useTranslations('button');
+
+  const ref = useRef<HTMLDivElement>(null);
+  const [isActive, setIsActive] = useState(false);
+
+  const { deleteWindow } = getStyle({ isActive });
```
Comment (Kyslytsia) [inline client/src/components/board-task-card/card/index.tsx:26]: All constants must be at the top
State below

**Review by Kyslytsia — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

File: client/src/components/board-task-card/card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import { Participants } from '@/components/participants';
+import { IMokUserCountProps } from '@/components/participants/mock-user';
+
+import { getStyle } from '../style';
+import { IMockCards } from '../mock';
+
+interface ICard {
+  id: string;
+  title: string;
+  data: IMockCards[];
```
Comment (Wolshebnik) [inline client/src/components/board-task-card/card/index.tsx:15]: wrong name

**Review by Wolshebnik — CHANGES_REQUESTED** [review]

File: client/src/app/[locale]/(provider)/manager/dashboards/board/[slug]/page.tsx [hunk 3]
```diff
@@ -1,5 +1,9 @@
+'use client';
```
Comment (Wolshebnik) [inline client/src/app/[locale]/(provider)/manager/dashboards/board/[slug]/page.tsx:1]: space

File: client/src/app/[locale]/(provider)/manager/dashboards/board/[slug]/page.tsx [hunk 4]
```diff
@@ -1,5 +1,9 @@
...
+import { useState } from 'react';
 import { Metadata } from 'next';
 
+import { TaskCard } from '@/components';
+import { mockCards } from '@/components/task-card/mock';
 import { BoardTitle } from './board-title';
```
Comment (Wolshebnik) [inline client/src/app/[locale]/(provider)/manager/dashboards/board/[slug]/page.tsx:7]: space

File: client/src/app/[locale]/(provider)/manager/dashboards/board/[slug]/page.tsx [hunk 5]
```diff
@@ -16,13 +20,19 @@ export async function generateMetadata({ params }: Props): Promise<Metadata> {
...
 const DashboardSlug = ({ params }: Props) => {
+  const [data, setData] = useState(mockCards);
   const titleContent =
     'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Laboriosam nam praesentium error natus voluptatum eos, explicabo quis incidunt quae delectus illo eius facere necessitatibus molestiae ipsam perspiciatis repellat eligendi vitae?';
 
   return (
     <>
       <BoardTitle titleText={titleContent} />
-
+      <TaskCard
```
Comment (Wolshebnik) [inline client/src/app/[locale]/(provider)/manager/dashboards/board/[slug]/page.tsx:30]: remove

File: client/src/components/tools-drop-menu/style.ts [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import { cn } from '@/utils';
+// import clsx from 'clsx';
```
Comment (Wolshebnik) [inline client/src/components/tools-drop-menu/style.ts:2]: remove

File: client/src/components/tools-drop-menu/style.ts [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+  animationOpen = 'openToolsMenu',
+  animationClose = 'closeToolsMenu',
+}: IGetStyleProps) => ({
+  toolsMenu: cn(
+    'border-swissCoffee absolute right-0 top-0 z-[10] flex w-[200px] flex-col justify-between gap-2 rounded-lg border bg-white px-3 py-2 shadow-boardCard translate-x-full shadow-lg',
+    className,
+    // {
+    //   'animate-openToolsMenu': opened,
+    //   'animate-closeToolsMenu': !opened,
+    // },
```
Comment (Wolshebnik) [inline client/src/components/tools-drop-menu/style.ts:23]: remove

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**